packages/commons/src/logging/loggerMiddleware.ts

```diff
@@ -96,7 +96,8 @@ export const loggerMiddleware = (serviceName: string) => () =>
 
 export const logger = getLogger();
 if (!parsedLoggerConfig.success) {
-  logger.info(
+  // eslint-disable-next-line no-console
+  console.log(
```

Comment:
Is this correct?

Reply:
yes, it's because in this case logger instance could not already created properly and could some problem, for now I suggest to use console for logging, and invest with more time with a specific task for logging instance improvement.